packages/test-data/css/_main/container.css

```diff
@@ -0,0 +1,148 @@
+.widget.discoverresults,
+.widget.repositoriesresults {
+  container-type: inline-size;
+}
+@container (max-width: 350px) {
+  .widget.discoverresults .cite .wdr-authors,
+  .widget.repositoriesresults .cite .wdr-authors {
+    display: none;
+  }
+}
+@container (min-width: 700px) {
+  .card h2 {
+    font-size: 2em;
+  }
+}
+@container sidebar (min-width: 700px) {
+  .card {
+    font-size: 2em;
+  }
+}
+@container (min-width: 60ch) {
+  .container:nth-child(odd) > article {
+    border: 1px solid grey;
+  }
+}
+@container size (min-width: 60ch) {
+  .article--post header {
+    grid-template-areas: "avatar name" "avatar headline";
+    align-items: start;
+  }
+  .article--post {
+    grid-template-areas: "header header" ". stats" ". content";
+    grid-auto-columns: 5rem 1fr;
+    column-gap: 1rem;
+  }
+  .article--post__title {
+    font-size: 1.75rem;
+  }
+}
+.card h2 {
+  container-type: inline-size;
+  margin: 0;
+  padding: 10px;
+}
+@container (min-width: 500px) {
+  .card h2 .card-content {
+    grid-template-columns: 1fr 2fr;
+    grid-template-rows: auto 1fr;
+    align-items: start;
+    column-gap: 20px;
+  }
+  .card h2 .card-content h2 {
+    padding: 0;
+    margin: 0.5em 0 0 0;
+  }
+}
+@container (width >= 500px) and (height >= 500px) {
```

Comment:
Oh this is great!  Does this add support for Media Queries Level 4 or did you only add Level 4 parsing for container queries?

Reply:
Ope, no, this doesn't add support. Seems like it would be trivial with these changes?